pysignalscope/scope.py

```diff
@@ -27,6 +33,68 @@
 class HandleScope:
     """Class to share scope figures in a special format, to keep labels, units and voltages belonging to a certain curve."""
 
+    # - private members ------------------------------------------------------------------------------
```

Comment:
what is the purpose writing these attribues to the class?
i thought, the target is to have a class that groups all the static methods?

Reply:
If this is used for the active plot, shouldn't it be in a separate class?

Reply:
Issue to discuss in a meeting. 
The approach of an additional class improves the structure. However short discussion shall be done before implementation.

Reply:
as discussed, will be changed